packages/adblocker/src/filters/network.ts

```diff
@@ -1273,6 +1387,26 @@ export default class NetworkFilter implements IFilter {
     return this.redirect || '';
   }
 
+  public hasRawOptions(): boolean {
+    return this.rawOptions !== undefined;
+  }
+
+  public isHtmlFilteringRule(): boolean {
+    return this.hasRawOptions() && this.rawOptions!.has('replace');
+  }
+
+  public getHtmlFilteringRegex(): [RegExp, string] | null {
+    if (!this.isHtmlFilteringRule()) {
+      return null;
+    }
+
+    const optionValue = this.rawOptions!.get('replace')!;
+    const [, rawRegexp, replacement, modifiers] = splitUnescaped(optionValue, '/');
+    const regexp = new RegExp(rawRegexp, modifiers);
+
+    return [regexp, replacement];
```

Comment:
If the filter doesn't need to deal with multiple html filtering filter modifiers, we may build regex and replacement at the parse time instead of runtime.